tests/unittests/ImageIO/ImageIOTest.mm

```diff
@@ -1233,12 +1233,12 @@ static CFURLRef getURLRefForOutFile(const wchar_t* filename) {
     int loopCount = 15;
     float delayTime = 0.5;
     NSDictionary* gifEncodeOptions = @{
```

Comment:
not a fan of the new formatting and did not expect you to be touching files outside of foundation tests given the status in your .md